google/import_cloudfunctions_function_test.go

```diff
@@ -0,0 +1,41 @@
+package google
```

Comment:
This whole file can be removed, since you're already testing import in the resource_ tests (thanks for doing that!)

Reply:
Done :)